ramls/holdingsrecordssource.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A holdings records source",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "The unique ID of the holdings records source; UUID",
+      "$ref": "uuid.json"
+    },
+    "name": {
+      "type": "string",
+      "description": "Name of the holdings records source"
+    },
+    "source": {
+      "type": "string",
+      "description": "The holdings records source"
+    }
```

Comment:
Should this be constrained by an enum to only allow `folio` and `local`, as I think is described in my comment on the issue that Charlotte agreed to?

Reply:
Is there an example of this approach? I'm not sure where to put the enum class within the package structure.

Reply:
I think I figured it out. Enum is a type available within the RAML spec.